### ISSUE TITLE
Can fadeOut animations have a delay?

### ISSUE THREAD

**Issue by kevando** [oblador/react-native-animatable#52]
I'm kinda goin crazy here

**Comment by oblador** [comment]
Yes, what code are you trying with?

**Comment by kevando** [comment]
I would like to display a text response, and have it fadeout 5 seconds later. And if the user does that action again, do show the same status and then fadeout 5 seconds later.

I did it using timeout, it just seemed like it would be part of this library.

**Comment by oblador** [comment]
If you use the declarative way then it's supported with the `delay` prop.

**Comment by kevando** [comment]
Can you provide an example of what that means? I tried a lot of things to get this to work.

**Comment by oblador** [comment]
There's plenty of examples – see the part of [the documentation about declarative usage](https://github.com/oblador/react-native-animatable#declarative-usage) or the [example project](https://github.com/oblador/react-native-animatable/blob/master/Example/app.js#L193).

**Comment by kevando** [comment]
Yeah it seems the delay only works when it animates from componentDidMount or wherever it initializes from. If I try to run that animation again, I don't see a delay.

**Comment by colorhaake** [comment]
@kevando, you can check [this](https://github.com/oblador/react-native-animatable/pull/44/files) to fix delay issue

**Comment by bberak** [comment]
Hi @kevando,

Not sure if this applies to you, but try using the iterationDelay prop - that sort of did the trick for me:

```
          <Animatable.View
            duration={1000}
            iterationDelay={300}
            iteration={1}
            animation={
              this.state.menuOpen ? "fadeIn" : "fadeOut"
            }
            useNativeDriver={true}
          >
           ...
          </Animatable.View>
```
